gob/gob.go

```diff
@@ -43,4 +44,7 @@ func init() {
 	gob.Register(plotter.XYZs{})
 	gob.Register(plotter.XYValues{})
 
+	// vg/draw.TextStyle
+	gob.Register(plot.DefaultTextHandler)
+	gob.Register(text.Plain{})
```

Comment:
`plot.DefaultTextHandler == text.Plain{}`; why do we register both?

Reply:
mainly for "forward coding" (in the perhaps remote case when we'd switch DefaultTextHandler to default to another implementation)

I can remove if you prefer.

Reply:
OK. That's fine.